src/controls/arrow.js

```diff
@@ -31,11 +39,11 @@ export default class Arrow extends Component {
   hasNext() {
```

Comment:
@chinrichs-godaddy : Do you think its ok to move `hasNext()` to index.js similar to nextSlide() and prevSlide();
And pass it down as a prop ?
This way it would be able to disable the button in the custom control component ?

Reply:
nvm, I see it varies based on `direction, infinite, numSlides, selectedIndex`